src/pages/elements/Clusters.js

```diff
@@ -1,61 +1,185 @@
-import React, { useState } from 'react';
-import { Row, Col, Card, CardHeading, CardBody, CardFooter, CardTitle } from 'patternfly-react';
+import React from 'react';
 import { Link } from 'react-router-dom';
-import { chunk } from 'lodash';
-import GridSearch from '../../components/GridSearch';
+import { Table } from 'patternfly-react';
 import { sortByName } from '../../components/Utils';
 
+function AppSREClustersTable({ clusters }) {
+  const headerFormat = value => <Table.Heading>{value}</Table.Heading>;
+  const cellFormat = value => <Table.Cell>{value}</Table.Cell>;
+  return (
+    <Table.PfProvider
+      striped
+      bordered
+      columns={[
+        {
+          header: {
+            label: 'Name',
+            formatters: [headerFormat]
+          },
+          cell: {
+            formatters: [
+              value => (
+                <>
+                  <Link
+                    to={{
+                      pathname: '/clusters',
+                      hash: value[1]
+                    }}
+                  >
+                    {value[0]}
+                  </Link>
+                  &nbsp;&nbsp;
+                  <a href={value[2]}>
+                    <i className="fa fa-desktop" />
+                  </a>
+                </>
+              ),
+              cellFormat
+            ]
+          },
+          property: 'name_path'
+        },
+        {
+          header: {
+            label: 'Description',
+            formatters: [headerFormat]
+          },
+          cell: {
+            formatters: [cellFormat]
+          },
+          property: 'description'
+        },
+        {
+          header: {
+            label: 'Version',
+            formatters: [headerFormat]
+          },
+          cell: {
+            formatters: [cellFormat]
+          },
+          property: 'version'
+        },
+        {
+          header: {
+            label: 'Channel',
+            formatters: [headerFormat]
+          },
+          cell: {
+            formatters: [cellFormat]
+          },
+          property: 'channel'
+        },
+        {
+          header: {
+            label: 'External ID',
```

Comment:
i'd use id over external id, as that is the language we speak with OCM/SREP

Reply:
good feedback, thanks!